src/common/providers/identity.ts

```diff
@@ -673,7 +678,8 @@ export function generateResponsePayload(
     return {};
   }
 
-  const { recaptchaPassed, ...formattedAuthResponse } = authResponse;
+  const { recaptchaActionOverride: recaptchaActionOverride, ...formattedAuthResponse } =
```

Comment:
Am I missing something with the destructuring assignment for `recaptchaActionOverride`, or can we just do `const { recaptchaActionOverride, ...formatedAuthResponse } = authResponse;` instead?